tests/lib/rook/ses.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2019 SUSE LINUX GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os
+import filecmp
+
+from tests.lib.common import execute
+from tests.lib.rook.base import RookBase
+
+logger = logging.getLogger(__name__)
+
+
+class RookCluster(RookBase):
+    def __init__(self, workspace, kubernetes):
+        super().__init__(workspace, kubernetes)
+        self.ceph_dir = os.path.join(
+            self.workspace.working_dir, 'rook', 'ceph')
+
+    def _get_rook_files(self):
+        execute(f"rsync -avr -e 'ssh -i {self.workspace.private_key}'"
+                f" sles@{self.kubernetes.hardware.masters[0].get_ssh_ip()}"
+                f":/usr/share/k8s-yaml/rook {self.workspace.working_dir}")
+
+    # TODO: DISCUSS how to handle registry.suse.com vs registry.suse.de
+    def _fix_yaml(self):
+        replacements = {
+            'suse.com': 'suse.de/devel/storage/7.0/containers',
+            '# ROOK_CSI_CEPH_IMAGE': 'ROOK_CSI_CEPH_IMAGE'
+        }
+        for root, dirs, files in os.walk(self.ceph_dir):
```

Comment:
Do we really want to unconditionally iterate over all files and all lines? I would prefer to be more specific about the files we want to change.

Reply:
Actually we don't know in advance which files will be called. It might depends on the tests you are going to run.
If we make the change for one, we'll have to make for all anyway.
Also, processing all of them makes it support any future file added.

Reply:
Is that ok @toabctl ?

Reply:
Works for me.